Cargo.toml

```diff
@@ -23,6 +23,7 @@ async-task = "4.4.0"
 concurrent-queue = "2.5.0"
 fastrand = "2.0.0"
 futures-lite = { version = "2.0.0", default-features = false }
+pin-project-lite = "0.2"
```

Comment:
Noting to myself that this is not a new dependency, since `futures-lite` imports it already.